tests/nginx.nix

```diff
@@ -348,5 +348,21 @@ in {
       assert_reachable(server4, "fe.local -6")
       assert_unreachable(server4, "fe.local -4")
       assert_unreachable(server4, "srv.local")
+
+    with subtest("[5] rate limiting connections"):
+      import re
+      # Running this against the other virtual hosts fails. I *think* this is
+      # because we have a "return 200" statement on the root location there
+      # which seems to short-circuit the rate limiting ... o_O
```

Comment:
The `return` statement is [documented](http://nginx.org/en/docs/http/ngx_http_rewrite_module.html#return) as:

> Stops processing and returns the specified code to a client

This wording does imply that there might be some kind of short-circuiting behaviour here.